docs/SrcCS.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<ruleset name="Loris Standard">
+    <description>The coding standard for the external src files</description>
+
+    <!-- Include the PEAR standard -->
+    <rule ref="PSR2"/>
```

Comment:
I think we still want all the other things that the LorisCS.xml adds on top of PEAR to be enforced for the "src" directory.. it would be better to just copy the file and change the PEAR ref to "PSR2"

Reply:
I added them below